### DISCUSSION TITLE
How to hire a mugger for all players in a session?

### DISCUSSION THREAD

**Discussion by Ujfetumsy** [YimMenu/YimMenu#3259] [Q&A]
how to hire a mugger for all players in a session? I was looking for some command or Lua script and found nothing. maybe someone knows how to do this? Perhaps someone has such a lua script?

**Comment by ghost** [comment]
we seemed to have a function like a x-mas mugger, but it disappeared somewhere or something

**Reply by Tayipc38** [reply]
> we seemed to have a function like a x-mas mugger, but it disappeared somewhere or something

It was a feature @maybegreat48 was testing and left in accidentally. Muggers are subject of the AM_GANG_CALL script iirc.
